SDVModTest/IconHandler.cs

```diff
@@ -28,7 +28,7 @@ private IconHandler()
         public Point GetNewIconPosition()
         {
             int yPos = Game1.options.zoomButtons ? 290 : 260;
-            int xPosition = (int)Tools.GetWidthInPlayArea() - 134 - 46 * _amountOfVisibleIcons;
+            int xPosition = (int) Tools.GetWidthInPlayArea() - 134 - 46 * _amountOfVisibleIcons;

```

Comment:
Casting is not part of the equation, it's directly applied to the one value. The space doesn't belong here.